config/crd/bases/tower.ansible.com_workflowtemplates.yaml

```diff
@@ -116,7 +116,6 @@ spec:
             - connection_secret
             - name
             - organization
-            - inventory
```

Comment:
This change could cause some conflicts on the CRD that may prevent older versions of the operator from being installed in other namespaces on the same cluster.  We should coordinate merging this with a new  apiVersion of the CRD.